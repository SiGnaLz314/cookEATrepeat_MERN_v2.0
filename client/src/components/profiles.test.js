import React from 'react';
import renderer from 'react-test-renderer';
import Profile from './profiles';
import { BrowserRouter } from 'react-router-dom';

const users=[
    'test', 'test2', 'test3'
]; 
const admin=false;
const loggedIn=true;



describe('Recipe-List tests', ()=> {
    test('Recipe-List snapshot test', () => {
        const component = renderer.create(
            <BrowserRouter>
                <Profile
                    users={users}
                    loggedIn={loggedIn}
                    admin={admin} />
            </BrowserRouter>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
