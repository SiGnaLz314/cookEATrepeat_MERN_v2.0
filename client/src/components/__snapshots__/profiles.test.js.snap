// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe-List tests Recipe-List snapshot test 1`] = `
<div>
  <h2>
    USERS:
  </h2>
  <table
    className="table"
  >
    <thead
      className="thead-light"
    >
      <tr>
        <th>
          User:
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
